### PR TITLE
add colorsys

### PR DESCRIPTION
**Pull request by tharvik**
Anything more to say?

### CODE REVIEW

File: stdlib/2and3/colorsys.pyi [hunk 1]
```diff
@@ -0,0 +1,10 @@
+# Stubs for colorsys
+
+from typing import Tuple
+
```
Comment (matthiaskramm) [inline stdlib/2and3/colorsys.pyi]: Please add the constants `ONE_SIXTH`, `ONE_THIRD`, `TWO_THIRD`. (All floats)

File: stdlib/2and3/colorsys.pyi [hunk 2]
```diff
@@ -0,0 +1,10 @@
...
+def rgb_to_yiq(r: float, g: float, b: float) -> Tuple[float, float, float]: ...
```
Comment (matthiaskramm) [inline stdlib/2and3/colorsys.pyi]: This function actually works with `complex`, too.

File: stdlib/2and3/colorsys.pyi (hunk 1 above)
Comment (tharvik) [inline stdlib/2and3/colorsys.pyi]: These are undocumented, seems internal (not in `__all__`, nor having description); it was added in fc55ed0 if really wanted.

File: stdlib/2and3/colorsys.pyi (hunk 2 above)
Comment (tharvik) [inline stdlib/2and3/colorsys.pyi]: As every function working with float should work with complex, right? And it is also undocumented, added in 04458e6.

Comment (matthiaskramm) [inline stdlib/2and3/colorsys.pyi]: Strictly speaking, not every code working with float works with complex. For example `x > 3.1` doesn't.

File: stdlib/2and3/colorsys.pyi [hunk 3]
```diff
@@ -13,3 +12,6 @@ def hsv_to_rgb(h: float, s: float, v: float) -> Tuple[float, float, float]: ...
 ONE_SIXTH = ...  # type: float
 ONE_THIRD = ...  # type: float
 TWO_THIRD = ...  # type: float
+
+# TODO undocumented, should only work with float
+def rgb_to_yiq(r: Union[float, complex], g: Union[float, complex], b: Union[float, complex]) -> Tuple[float, float, float]: ...
```
Comment (matthiaskramm) [inline stdlib/2and3/colorsys.pyi]: mypy allows you to use the shortcut `complex` for `Union[float, complex]`, I think.

Comment (tharvik) [inline stdlib/2and3/colorsys.pyi]: Indeed, seems to work, updated in f9413c4.

File: stdlib/2and3/colorsys.pyi (hunk 2 above)
Comment (tharvik) [inline stdlib/2and3/colorsys.pyi]: True; then mypy is making a shortcut by having `complex` being `Union[complex, float]`.

File: stdlib/2and3/colorsys.pyi [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
+def rgb_to_hsv(r: float, g: float, b: float) -> Tuple[float, float, float]: ...
+def hsv_to_rgb(h: float, s: float, v: float) -> Tuple[float, float, float]: ...
+
+# TODO undocumented
+ONE_SIXTH = ...  # type: float
+ONE_THIRD = ...  # type: float
+TWO_THIRD = ...  # type: float
+
+# TODO undocumented, should only work with float
+def rgb_to_yiq(r: complex, g: complex, b: complex) -> Tuple[float, float, float]: ...
```
Comment (matthiaskramm) [inline stdlib/2and3/colorsys.pyi]: Actually, the result is complex, too, if the input is complex.

It's probably not worth it to model that with an `@overload` or `TypeVar`. I'm fine with it if you just change it back to float. Nobody is *supposed* to call this function with a complex, anyway. Sorry for the back and fro.

Comment (tharvik) [inline stdlib/2and3/colorsys.pyi]: Don't worry, it's what PR are for.
Back to float then (mypy should also help to see if someone use the stdlib in a weird/unsupported way).

**Pull request merged**